ui/src/app/base/components/SegmentedControl/SegmentedControl.tsx

```diff
@@ -0,0 +1,43 @@
+import type { HTMLAttributes } from "react";
+
+import classNames from "classnames";
+
+type Segment<V> = {
+  title: string;
```

Comment:
I wonder if `label` would be more appropriate (and consistent with `label` `value` pair that's already used when dealing with collection of options in react-components `Select`).

`title` can be confused with the `title` attribute.